src/client/app/components/admin/UsersDetailComponent.tsx

```diff
@@ -52,8 +53,8 @@ export default function UserDetailComponent(props: UserDisplayComponentProps) {
 						<thead>
 							<tr>
 								<th> Email </th>
-								<th> Role </th>
-								<th> Action </th>
+								<th> <FormattedMessage id='role'/> </th>
```

Comment:
The translations already have "e-mail" so maybe do "Email" above.

Reply:
Found an issue with e-mail only being available for Spanish in the data.json translation file, while french and english had email instead. Fixed it to have email for all 3, and included it in the following code.